### PR TITLE
[debian] Separate debian packaging

### PR DESCRIPTION
**Pull request by gichan-jang**
- Separate single/pipeline debian package
 - Change library name (capi-nnstreamer-* -> ml-inference-*)

Signed-off-by: gichan <gichan2.jang@samsung.com>

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -18,17 +24,45 @@ Description: Machine Learning API common
...
-Depends: nnstreamer, ${shlibs:Depends}, ${misc:Depends}
+Depends: ml-inference-single-api, ml-inference-pipeline-api, ${shlibs:Depends}, ${misc:Depends}
 Description: Machine Learning Inference API
  This package provides native API set to easily construct a data stream pipeline with neural networks.
 
 Package: ml-inference-api-dev
 Architecture: any
 Multi-Arch: same
-Depends: ml-inference-api, ml-api-common-dev, ${shlibs:Depends}, ${misc:Depends}
+Depends: ml-inference-single-api-dev, ml-inference-pipeline-api-dev, ${shlibs:Depends}, ${misc:Depends}
```
Comment (jaeyun-jung) [inline debian/control:65]: dependency to ml-inference-api

File: debian/control [hunk 2]
```diff
@@ -10,6 +10,12 @@ Build-Depends: gcc-9 | gcc-8 | gcc-7 | gcc-6 | gcc-5 (>=5.4),
 Standards-Version: 3.9.6
 Homepage: https://github.com/nnstreamer/api
 
+Package: ml-api-common
+Architecture: any
+Multi-Arch: same
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Description: Common utility functions for Tizen Machine Learning API
```
Comment (jaeyun-jung) [inline debian/control:17]: suggestion:
Common utility functions for Machine Learning API

File: debian/control (hunk 1 above)
Comment (gichan-jang) [reply debian/control]: Updated. Thanks!

**Review by jaeyun-jung — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**